static/css/tools/scatter.scss

```diff
@@ -140,6 +140,7 @@
 #enclosed-place-type {
   height: 100%;
   font-size: 1rem;
+  width: 225px;
```

Comment:
nit: for a followup - we could prefer to use a more scalable unit, like rem for this (to accommodate large fonts, etc).

Reply:
Ok, will do this in a followup